UIcode/server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
```

Comment:
There is no mention on how to start this server on the README. Can you please add documentation there and how it relates with the client?

Reply:
This is not needed anymore, I will delete it